src/Analyzers/Core/CodeFixes/ImplementInterface/IImplementInterfaceInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.ImplementInterface;
+
+internal interface IImplementInterfaceInfo
```

Comment:
was in a file related to implement-interface, which itself hasn't moved to this layer *yet*.  that will be in followup pr.

Reply:
I thought implement interface required a VS UI dialog - is that necessary to move to the code style layer?  Is there any SDK tool that could use it?